client/components/QuestionsAndAnswers/QAndAComponents/Helpful.jsx

```diff
@@ -3,27 +3,34 @@ import { makeStyles, Grid, Typography } from '@material-ui/core';
 import axios from 'axios';
 
 const useStyles = makeStyles((theme) => ({
-  underlined: {
-    textDecoration: 'underline'
+  yes: {
+    textDecoration: 'underline',
+    cursor: 'pointer'
   }
 }));
 
-const Helpful = ({ helpfulness }) => {
+const Helpful = ({ helpfulness, questionId, answerId }) => {
   const classes = useStyles();
   const [helpful, setHelpful] = useState(helpfulness);
   const [clicked, setClicked] = useState(false);
 
   const handleClick = () => {
-    if (!clicked) {
-      setHelpful(helpfulness + 1);
-      setClicked(true);
+    if (clicked) {
+      return;
     }
+    if (questionId === 'NA') {
+      axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hratx/qa/answers/${answerId}/helpful`)
+    } else if (answerId === 'NA') {
+      axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hratx/qa/questions/${questionId}/helpful`)
+    }
+    setHelpful(helpfulness + 1);
```

Comment:
I'm wondering if lines 26 and 27 should be moved into a .then block after the put request?  I think the way it is now, state will be updated regardless if the put request is successful or not, whereas if you put it in .then() you would only be updating state if the request was successful.